front-end-side/src/Pages/Data.css

```diff
@@ -173,25 +173,50 @@ table.data_table {
         background: #373f4d;
         border-radius: 0 0 10px 10px;
         margin-top: 5px;
-        height: 300px;
-        padding: 20px;
+        padding: 20px 5.208vw 10px;
     }
 
+.recharts-wrapper, .recharts-wrapper svg.recharts-surface {
+    width: 100% !important;
+    height: auto !important;
+}
+
 
 @media only screen and (max-width: 1200px) {
     #statistics #cards {
         grid-template-columns: 1fr 1fr;
     }
 }
 
+@media only screen and (max-width: 900px) {
+    table.data_table td:nth-of-type(3) {
+        max-width: 30vw !important;
+    }
+}
+
 @media only screen and (max-width: 800px) {
     #statistics #cards {
         grid-template-columns: 1fr 1fr;
     }
 }
 
 @media only screen and (max-width: 700px) {
-    #statistics #cards {
+    #statistics #cards, #statistics-graph {
         grid-template-columns: 1fr;
     }
+
+    #statistics-graph {
+        grid-row-gap: 20px;
+    }
+
+    table.data_table td:nth-of-type(3) {
+        max-width: 20vw!important;
+    }
+}
+
+@media only screen and (max-width: 600px) {
+    table.data_table td.edit, table.data_table td.move, table.data_table td.delete, table.data_table th.last {
+        display: none;
```

Comment:
Why did we stop displaying those at 600px?

Reply:
It is not so neccessary to see them all the time, especialy when we don't have much space (in mobile view). In the future we will make those buttons apear after clicking on a particular record.